data/settings/src/main/proto/com/google/jetpackcamera/settings/jca_settings.proto

```diff
@@ -44,4 +45,6 @@ message JcaSettings {
   DynamicRange dynamic_range_status = 13;
   repeated DynamicRange supported_dynamic_ranges = 14;
   LensFacing default_lens_facing = 15;
+  TargetFrameRate target_frame_rate = 16;
```

Comment:
Is there a reason this can't just be an int32 rather than an enum? It's possible some devices might not support 15fps, for instance, and may support something like 12 fps. It will be much easier to just keep things as ints so that we can more easily support cases like this in the future.

Reply:
you're right, this could just be an int. i think when i was originally writing the datastore values i wanted to keep it congruent with the data model used for `CameraAppSettings`. 

I think alternatively, since target frame rate is a range, the datastore can hold onto two ints to represent the range's upper and lower values. this isn't necessary right now because the implementation is only looking at fixed values 15, 30, or 60 fps, but I think it'd still be something to consider if we'd ever wanted to dynamically change the options available on the setting.